fortran/finufftfort.cpp

```diff
@@ -78,19 +78,17 @@ extern "C" {
 #endif
   
 // --------------------- guru interface from fortran ------------------------
-void FINUFFT_MAKEPLAN_(int *type, int *n_dims, BIGINT *n_modes, int *iflag, int *n_transf, FLT *tol, FINUFFT_PLAN **plan, nufft_opts *o, int *ier)
+void FINUFFT_MAKEPLAN_(int *type, int *n_dims, BIGINT *n_modes, int *iflag, int *n_transf, FLT *tol, FINUFFT_PLAN *plan, nufft_opts *o, int *ier)
```

Comment:
Would suggest calling plan -> pplan or planptr here and throughout. (doing same for M, etc would be overkill).

Reply:
Agreed.